### DISCUSSION TITLE
Tailwind don't work on angular 11

### DISCUSSION THREAD

**Discussion by habib6bd** [tailwindlabs/tailwindcss#9949] [Help]
I follow the guidelines for implementation tailwind with cli on my angular project But it don't work at all. Any suggestions for me?

**Comment by wongjn** [comment]
Nope! Could we see your project so we can look at what might be incorrect?

**Reply by habib6bd** [reply]
My github project link:
https://github.com/habib6bd/tailwind-angular

**Reply by wongjn** [reply]
I added a `postcss.config.js` and it seems to work now. You can create a sample one by running:
```sh
npx tailwindcss init -p
```

**Reply by habib6bd** [reply]
Now it's working properly. I'm very glad having your great support.
May be the earlier version of angular have some dependency with tailwind. Latest version's are okay.
